### PR TITLE
TM-3699: Edit Agenda from Employee Agenda Search 

### PR DESCRIPTION
**Pull request by pveland621**
ACs:
1. Verify clicking pencil edit icon takes user to Create Agenda Item screen with correct userRole and perdet
2. Verify both row and card views

Card View:
image

Row View:
image

### CODE REVIEW

File: src/sass/_employeeSearch.scss [hunk 1]
```diff
@@ -254,6 +254,11 @@
       }
     }
   }
+  .employee-card-status {
+    display: flex;
+    justify-content: space-between;
+    width: 55%;
```
Comment (andrews-john) [inline src/sass/_employeeSearch.scss:260]: Is there a better way to style this than using width (genuinely asking)

Comment (pveland621) [reply src/sass/_employeeSearch.scss:260]: There honestly might be, let me look a little more into other options. This is just the easiest way for me to try and get css to work with me lol

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx [hunk 2]
```diff
@@ -104,7 +103,12 @@ const EmployeeAgendaSearchCard = ({ isCDO, result, showCreate, viewType }) => {
         <div className="employee-card-data-point">
           <FA name="sticky-note-o" />
           <dt>Agenda Status:</dt>
-          <dd>{agendaStatus}</dd>
+          <dd className="employee-card-status">
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:106]: you'll want to put Edit functionality behind a boolean - we will eventually hook that boolean into "is this Agenda Item editable atm" criteria once we set that up

File: src/sass/_employeeSearch.scss (hunk 1 above)
Comment (elizabeth-jimenez) [reply src/sass/_employeeSearch.scss:260]: You can do something like this

```
  <div className="employee-card-data-point">
    <FA name="sticky-note-o" />
    <dt>Agenda Status:</dt>
    <dd>{agendaStatus}</dd>
    <Link to={`/profile/${userRole}/createagendaitem/${perdet}`} className="sophie">
      <FA name="pencil" />
    </Link>
  </div>
```

it simplifies the structure. Then on `sophie`🐈‍⬛ you'll throw a margin-left

Comment (pveland621) [reply src/sass/_employeeSearch.scss:260]: Change made - my thought process was 'mess with specific pixel values as little as possible due to resizing issues' so I thought using a flexbox with percentage width of the parent would be better for scalability

File: src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx (hunk 2 above)
Comment (pveland621) [reply src/Components/Agenda/EmployeeAgendaSearchCard/EmployeeAgendaSearchCard.jsx:106]: Set up new prop variable to display/hide edit button

**Review by andrews-john — APPROVED** [review]
Approved

**Pull request merged**